tox.ini

```diff
@@ -17,10 +17,14 @@ conda_deps =
     pybind11>=2.3.0
     ninja
     cmake
-    compilers
-
-conda_channels =
-    conda-forge
+    conda-forge::gcc_linux-64==7.3.0
+    conda-forge::gcc_impl_linux-64==7.3.0
+    conda-forge::gxx_linux-64==7.3.0
+    conda-forge::gxx_impl_linux-64==7.3.0
+    libgcc-ng==7.3.0
+    libstdcxx-ng==7.3.0
+    binutils_linux-64==2.31.1
+    binutils_impl_linux-64==2.31.1
```

Comment:
Great, thanks a lot for the help! Don't you think that would be interesting if we use `>=` to keep track updates in the compilers?

Reply:
Yes, let's see first if this works first, its just a guess :)
If it works, we change to >=